ui/mods/mod_dynamic_perks/hooks/screens/character/modules/character_screen_right_panel/character_screen_perks_module.js

```diff
@@ -55,6 +58,37 @@ CharacterScreenPerksModule.prototype.initPerkTree = function (_perkTree, _perksU
 	}
 };
 
+DynamicPerks.Hooks.attachEventHandler = CharacterScreenPerksModule.prototype.attachEventHandler;
+CharacterScreenPerksModule.prototype.attachEventHandler = function(_perk)
+{
+	DynamicPerks.Hooks.attachEventHandler.call(this, _perk);
+	var self = this;
+
+	_perk.Container.on('mouseenter.dynamicperks focus.dynamicperks' + CharacterScreenIdentifier.KeyEvent.PerksModuleNamespace, null, this, function (_event)
+	{
+		self.mContainer.find("[dynamicperksperkgroupid='" + _perk.PerkGroupID + "'] .dynamicperks-image-overlay").css("border", "2px solid rgba(" + MSU.getSettingValue("mod_dynamic_perks", "pergroup_highlight_color") + ")");
+	});
+
+	_perk.Container.on('mouseleave.dynamicperks blur.dynamicperks' + CharacterScreenIdentifier.KeyEvent.PerksModuleNamespace, null, this, function (_event)
+	{
+		self.mContainer.find("[dynamicperksperkgroupid='" + _perk.PerkGroupID + "'] .dynamicperks-image-overlay").css("border", "none");
+	});
+}
+
+DynamicPerks.Hooks.attachEventHandler = CharacterScreenPerksModule.prototype.removePerksEventHandler;
+CharacterScreenPerksModule.prototype.removePerksEventHandler = function (_perkTree)
+{
+	DynamicPerks.Hooks.attachEventHandler.call(this, _perkTree);
```

Comment:
`attachEventHandler` vs `removePerksEventHandler`, in fact this is certainly bugged

Reply:
Oops